### PR TITLE
Update whats-new-in-v11-0.md

### PR DESCRIPTION
**Pull request by itspooya**
Inconsistency in https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v11-0/ and https://grafana.com/docs/grafana/latest/explore/explore-metrics/ Explore Metrics is currently in private preview. Grafana Labs offers support on a best-effort basis, and breaking changes might occur prior to the feature being made generally available. 

This mismatch is because the feature is in private preview in version 10.4 and in public preview in version 11. However, the 11-preview What's new page points to the version 10.4 documentation for the feature. This PR updates the docs link to point to the version 11 docs. These docs have also been updated to reflect the correct release stage on PR #86390

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v11-0.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Use full URLs for links. When linking to versioned docs, replace the version wit
 
 <!-- #proj-datatrails-dev, PM: Jay Goodson, Engineering: Darren Janeczek, André Pereira, Design: Catherine Gui -->
 
-_Public preview in all editions of Grafana_
+_Private preview in all editions of Grafana_
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v11-0.md:57]:
```suggestion
_Public preview in all editions of Grafana_
```

**Comment by imatwawana** [comment]
@itspooya - Thanks for bringing this to our attention and for the contribution! I've updated the description and content of the PR with a fix that deals with the underlying problem. We'll get this merged shortly!

**Review by imatwawana — APPROVED** [review]

**Comment by itspooya** [comment]
Hi,
any updates here?
@imatwawana

**Comment by imatwawana** [comment]
Apologies! I'm not sure why this fell off my radar. Looking at this now.

**Comment by imatwawana** [comment]
This change is no longer needed since the v11.0 docs are now the latest version and the version interpolation syntax makes sense now. So I'm going to close this PR, but apologies for missing this back in April.

**Pull request closed without merging**